AnimateQuotes.js

```diff
@@ -29,46 +28,65 @@ export default function AnimateQuotes(props) {
         delay: QUOTES_IN_DELAY,
         useNativeDriver: true,
       }),
-    ]).start();
-    const endOfAnim = setTimeout(() => {
```

Comment:
setTimeout removed in favor of Animated start() method callback function for the same logic in order to make state deterministic.